### PR TITLE
Add Fortran and Shell support

### LINKED ISSUE #3: Support for Shell and Fortran
#### Is your feature request related to a problem? Please describe.

> _A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]_

Shell and Fortran languages should be supported, like in Docker-CAT.

#### Describe the solution you'd like

> _A clear and concise description of what you want to happen._

This image should be able to analyze shell and fortran projects.

#### Describe alternatives you've considered

> _A clear and concise description of any alternative solutions or features you've considered._

### PR DESCRIPTION
**Pull request by ThomasCluzel**
## Proposed changes

> _Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue._

This PR brings support for Fortran 77, Fortran 90 and Shell.

## Types of changes

What types of changes does your code introduce to this project?
> _Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Issues closed by changes

> _List here all issues closed by your changes. Use a list of items like `- [x] Close #0`_

- [x] Close #3 

## Checklist

> _Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I agree with the [CODE OF CONDUCT](CODE_OF_CONDUCT.md)
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tests/java.bash [hunk 1]
```diff
@@ -4,102 +4,14 @@
...
-           -v "$(pwd)/.sonarcache:/opt/sonar-scanner/.sonar/cache" \
-           lequal/sonar-scanner \
-              -Dsonar.projectBaseDir=/usr/src/tests/java \
-            2>&1)
-echo -e "$output"
+. tests/functions.bash
 
-# Make sure all non-default Java plugins were executed by the scanner
-expected_outputs=(
+sensors=(
```
Comment (ThomasCluzel) [inline tests/java.bash:9]: Codacy found an issue: [sensors appears unused. Verify use (or export if used externally).](https://app.codacy.com/gh/lequal/sonar-scanner/pullRequest?prid=5967771)

**Comment by ThomasCluzel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/lequal/sonar-scanner/pullRequest?prid=5967771&bid=19566766)

File: tests/fortran.bash [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# User story:
+# As a user of this image, I want to analyze a fortran project
+# so that I can see its level of quality on the SonarQube server.
+
+. tests/functions.bash
+
+sensors=(
```
Comment (ThomasCluzel) [inline tests/fortran.bash:9]: Codacy found an issue: [sensors appears unused. Verify use (or export if used externally).](https://app.codacy.com/gh/lequal/sonar-scanner/pullRequest?prid=5967771)

File: tests/shell.bash [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# User story:
+# As a user of this image, I want to analyze a shell project
+# so that I can see its level of quality on the SonarQube server.
+
+. tests/functions.bash
+
+sensors=(
```
Comment (ThomasCluzel) [inline tests/shell.bash:9]: Codacy found an issue: [sensors appears unused. Verify use (or export if used externally).](https://app.codacy.com/gh/lequal/sonar-scanner/pullRequest?prid=5967771)

File: Dockerfile [hunk 4]
```diff
@@ -4,3 +4,5 @@ FROM sonarsource/sonar-scanner-cli:4.4
 LABEL maintainer="LEQUAL"
 
 # TODO: install all tools available in Docker-CAT
+
+MAINTAINER Benoît and Thomas
```
Comment (ThomasCluzel) [inline Dockerfile:8]: Codacy found an issue: [MAINTAINER is deprecated](https://app.codacy.com/gh/lequal/sonar-scanner/pullRequest?prid=5967771)

**Comment by ThomasCluzel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/lequal/sonar-scanner/pullRequest?prid=5967771&bid=19566766)

**Comment by ThomasCluzel** [comment]
Problems with Codacy analysis are being investigated

**Comment by ThomasCluzel** [comment]
Anyone who commits to this repo should have his/her code analyzed by Codacy.

**Comment by ThomasCluzel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/lequal/sonar-scanner/pullRequest?prid=5967771&bid=19566766)

File: Dockerfile [hunk 5]
```diff
@@ -1,6 +1,8 @@
-# This image is based on the official sonar-scanner image
+ # This image is based on the official sonar-scanner image
 FROM sonarsource/sonar-scanner-cli:4.4
 
 LABEL maintainer="LEQUAL"
 
 # TODO: install all tools available in Docker-CAT
+
+MAINTAINER Benoît and Thomas
```
Comment (ThomasCluzel) [inline Dockerfile:8]: Codacy found an issue: [MAINTAINER is deprecated](https://app.codacy.com/gh/lequal/sonar-scanner/pullRequest?prid=5967771)

**Comment by ThomasCluzel** [comment]
Since the test with Codacy was conclusive, I reset the branch to the last real commit.

**Review by begarco — APPROVED** [review]

**Pull request merged**